examples/dgmulti_2d/elixir_navier_stokes_convergence.jl

```diff
@@ -0,0 +1,206 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the ideal compressible Navier-Stokes equations
+
+get_Re() = 100
+get_Pr() = 0.72
+
+equations = CompressibleEulerEquations2D(1.4)
+# Note: If you change the Navier-Stokes parameters here, also change them in the initial condition
+# I really do not like this structure but it should work for now
+equations_parabolic = CompressibleNavierStokesEquations2D(equations, Reynolds=get_Re(), Prandtl=get_Pr(),
+                                                          Mach_freestream=0.5)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+dg = DGMulti(polydeg = 3, element_type = Tri(), approximation_type = Polynomial(),
+             surface_integral = SurfaceIntegralWeakForm(flux_lax_friedrichs),
+             volume_integral = VolumeIntegralWeakForm())
+
+top_bottom(x, tol=50*eps()) = abs(abs(x[2]) - 1) < tol
+is_on_boundary = Dict(:top_bottom => top_bottom)
+mesh = DGMultiMesh(dg, cells_per_dimension=(16, 16); periodicity=(true, false), is_on_boundary)
+
+# Define initial condition
+# Note: If you change the parameters here, also change it in the corresponding source terms
+function initial_condition_navier_stokes_convergence_test(x, t, equations)
```

Comment:
Don't we ususally put these basic convergence tests into the `equations` folder (i.e., the IC and the source terms) to have it readily available for convtests when developing new methods, optimizing the implementation etc.?

Reply:
I would like that as well. I can move those files around later today.

Reply:
Resolved: we will keep the convergence tests in the elixirs until the issue with parameters (e.g., needing to define `reynolds_number() = ...` in the elixir) is resolved. cc: @ranocha